src/openforms/js/components/admin/form_design/Editor.js

```diff
@@ -9,14 +9,20 @@ const TinyMCEEditor = ({content, onEditorChange}) => {
   const editorRef = useRef(null);
   const tinyMceUrl = useContext(TinyMceContext);
   const intl = useIntl();
+  const useDarkMode = window.matchMedia('(prefers-color-scheme: dark)').matches;
```

Comment:
can you add a TODO for Django 4.2 - they now use explicit theme names rather than the media query approach: https://github.com/django/django/blob/main/django/contrib/admin/static/admin/css/dark_mode.css#L36